### PR TITLE
Add financial aid application form

### PR DESCRIPTION
**Pull request by inglesp**

### CODE REVIEW

**Comment by inglesp** [comment]
Addresses https://github.com/PyconUK/pyconuk-2016-internaldocs/issues/54 and https://github.com/PyconUK/pyconuk-2016-internaldocs/issues/60.

**Comment by inglesp** [comment]
Assigning to @Lukasa for review.

File: content/financial-aid.html [hunk 1]
```diff
@@ -3,6 +3,8 @@
 title: Financial Aid
 ---
 
+<iframe src="https://docs.google.com/forms/d/1JWdkK4pP-USm3duQ-mXdQ6rYEp05lrmqMQ0sSCA5JXw/viewform?embedded=true" width="760" height="800" frameborder="0" marginheight="0" marginwidth="0">Loading...</iframe>
+
```
Comment (Lukasa) [inline content/financial-aid.html:13]: Do we feel happy about embedding here?

Comment (inglesp) [inline content/financial-aid.html:13]: That's what I thought you'd suggested at https://github.com/PyconUK/pyconuk-2016-internaldocs/issues/60#issuecomment-214671406.  Happy to change to a link though, and even happier for you to do it!

Comment (Lukasa) [inline content/financial-aid.html:13]: Oh, I see, not quite what I meant. I think a link might work better: iframes are a bit gross. =P

Comment (Lukasa) [inline content/financial-aid.html:13]: I'll make the change though.

**Comment by snim2** [comment]
There is a TODO for this on the code4research page BTW

File: content/financial-aid.html (hunk 1 above)
Comment (inglesp) [inline content/financial-aid.html:13]: I want to get this out of the way, so I'll make the change now.

**Comment by Lukasa** [comment]
Ok, I've updated this. Now that I've touched it we need to tag another reviewer: @ghickman?

**Comment by Lukasa** [comment]
@snim2 Is there? I see a TODO for the ticket form, not the fin-aid form.

**Comment by snim2** [comment]
@Lukasa sorry, you are quite right!

**Comment by Lukasa** [comment]
Cool, in this case we can go. @snim2 as you're already distracted by this issue, wanna review and merge? ;)

**Comment by snim2** [comment]
Sure...

File: content/financial-aid.html [hunk 2]
```diff
@@ -3,6 +3,14 @@
 title: Financial Aid
 ---
 
+<h2>Apply Now</h2>
+
+<p>
+    To apply for financial aid, please fill out and submit <a "https://docs.google.com/forms/d/1JWdkK4pP-USm3duQ-mXdQ6rYEp05lrmqMQ0sSCA5JXw/viewform?embedded=true">this form</a>.
```
Comment (snim2) [inline content/financial-aid.html]: `this form` doesn't appear as a hyperlink for me, you are missing the `href=` in the anchor tag. Also, you have `embedded=true` in the URL, but I'm not sure you are actually embedding the form?

File: content/financial-aid.html [hunk 3]
```diff
@@ -57,9 +65,9 @@
 <h2>Applying</h2>
 
 <p>
-    This year we are running our financial aid process alongside our ticket
-    sales, using the same service. This allows us to centralise our data
-    collection.
+    To apply for financial aid, please complete the form at the top of this
+    page.  We'll be in contact when we've decided the status of your grant
+    application.  The deadline for financial aid applications is June 28th.
```
Comment (snim2) [inline content/financial-aid.html:70]: Why not link to the form here, as well as at the top of the page, to save the reader that extra bit of scrolling?

**Comment by snim2** [comment]
OK, have reviewed...

File: content/financial-aid.html (hunk 3 above)
Comment (Lukasa) [inline content/financial-aid.html:70]: Done.

File: content/financial-aid.html (hunk 2 above)
Comment (Lukasa) [inline content/financial-aid.html]: Done.

File: content/cfp.html [hunk 4]
```diff
@@ -80,7 +80,7 @@
 <p>We’ll publish the list of selected talks as soon as we can after the deadline.</p>
 
 <h2 id="financial-support">Financial support for speakers</h2>
-<p>Grants to assist speakers on low budgets will be available, as well as free and discounted tickets. They'll be allocated by the committee. Any speaker who needs help will receive it. Apply for a grant until the end of 10th May.</p>
+<p>Grants to assist speakers on low budgets will be available, as well as free and discounted tickets. They'll be allocated by the committee. Any speaker who needs help will receive it. <a href="/financial-aid/">Click here</a> to apply for financial aid. Financial aid applications close when the CFP closes.</p>
 
 <p>The grants committee will be especially interested in receiving applications from people belonging to under-represented demographic groups.</p>
```
Comment (snim2) [inline content/cfp.html]: Line 87 still contains a `TODO` which can be removed

**Comment by snim2** [comment]
OK, one small issue then I think we can merge.

File: content/cfp.html (hunk 4 above)
Comment (Lukasa) [inline content/cfp.html]: Done. =)

**Pull request merged**